File: youtube_playlist_randomizer/playlistitem.py
class PlayListItem:
    """
    PlayListItem object
    """

    def __init__(
        self,
        id: str,
        title: str,
        publishedAt: str,
        channelId: str,
        description: str,
        videoId: str,
    ) -> None:
        """
        PlayListItem object

        Args:
            id (str): playlist object
            title (str): title
            publishedAt (str): publishedAt
            channelId (str): channelId
            description (str): description
            videoId (str): videoId
        """
        self.id = id
        self.title = title
        self.publishedAt = publishedAt
        self.channelId = channelId
        self.description = description
        self.videoId = videoId

File: internal/models/playlistitem.go
package models

type PlayListItem struct {
	ID           string
	Title        string
	PublishedAt  string
	ChannelID    string
	ChannelTitle string
	Description  string
	VideoID      string
	ThumbnailURL string
}

func NewPlayListItem(id, title, publishedAt, channelID, channelTitle, description, videoID, thumbnailURL string) PlayListItem {
	return PlayListItem{
		ID:           id,
		Title:        title,
		PublishedAt:  publishedAt,
		ChannelID:    channelID,
		ChannelTitle: channelTitle,
		Description:  description,
		VideoID:      videoID,
		ThumbnailURL: thumbnailURL,
	}
}
